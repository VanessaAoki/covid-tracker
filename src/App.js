import React from 'react';
import {
  BrowserRouter as Router,
  Route,
  Routes,
  Navigate,
} from 'react-router-dom';
import MainPage from './components/Main';
import SecondPage from './components/Details';
import Footer from './components/Footer';
import Header from './components/Header';

const App = () => (
  <Router>
    <div>
      <Header />
      <div>
        <Routes>
          <Route path="/" element={<Navigate to="/main" />} />
          <Route path="/main" element={<MainPage />} />
          <Route path="/details" element={<SecondPage />} />
        </Routes>
      </div>
      <Footer />
    </div>
  </Router>
);

export default App;
