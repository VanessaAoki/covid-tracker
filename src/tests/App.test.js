import React from 'react';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import App from '../App';

describe('Home page renders as required', () => {
  it('Home UI matches the snapshot', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
