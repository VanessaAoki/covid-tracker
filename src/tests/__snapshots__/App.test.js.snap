// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page renders as required Home UI matches the snapshot 1`] = `
<div>
  <div />
  <footer
    className="footer"
  >
    **Data gathered from
     
    <a
      href="https://covid19tracking.narrativa.com/index_en.html"
    >
      Narrativa API
    </a>
  </footer>
</div>
`;
